### PR TITLE
[UI] Import Modal update and reference to Extensions 

### PR DESCRIPTION
**Pull request by Chadha93**
Signed-off-by: Gaurav Chadha <gauravchadha1676@gmail.com>
**Description**
Import Modal update and reference to Extensions keeping the UX as desired.

This PR fixes #
in reference to https://github.com/layer5labs/meshery-extensions/pull/1087

**Notes for Reviewers**

https://user-images.githubusercontent.com/65453826/182917825-0a49230b-e854-4ecd-8976-83cc9d18211b.mp4

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MesheryApplications.js [hunk 1]
```diff
@@ -783,7 +802,18 @@ function MesheryApplications({
         <div className={classes.topToolbar} >
           {!selectedApplication.show && (applications.length>0 || viewType==="table") && <div className={classes.createButton}>
             <div>
-              <UploadImport supportedTypes={types} isApplication = {true} aria-label="URL upload button" handleUrlUpload={urlUploadHandler} handleUpload={uploadHandler} configuration="Application"  />
+              <Button
+                aria-label="Add Pattern"
```
Comment (Chadha93) [inline ui/components/MesheryApplications.js:806]:
```suggestion
                aria-label="Add Application"
```

File: ui/components/Filters.js [hunk 2]
```diff
@@ -694,7 +713,18 @@ function MesheryFilters({ updateProgress, enqueueSnackbar, closeSnackbar, user,
         <div className={classes.topToolbar} >
           {!selectedFilter.show && (filters.length>0 || viewType==="table") && <div className={classes.createButton}>
             <div>
-              <UploadImport supportedTypes="null" aria-label="URL upload button" handleUrlUpload={urlUploadHandler} handleUpload={uploadHandler} configuration="Filter" />
+              <Button
+                aria-label="Add Pattern"
```
Comment (Chadha93) [inline ui/components/Filters.js:717]:
```suggestion
                aria-label="Add Filter"
```

**Comment by Harshit-Dandriyal** [comment]
woah, So, you Separated <UploadImport> and used buttons instead of that, Great but it's creating redundancy in code, I mean we should at least make sure that these buttons don't get repeated. I suggest making a component at least for buttons as they are even also similar in design.

**Comment by Chadha93** [comment]
> woah, So, you Separated UploadImport and used buttons instead of that, Great but it's creating redundancy in code, I mean we should at least make sure that these buttons don't get repeated. I suggest making a component at least for buttons as they are even also similar in design.

Yes, I'm with you, this was a fix we could go, for now, in the configurations code (filter/apps/patterns) there are lots of more lines of code causing redundancy. It is a very long process to remove all of it. We need to make sure all of these should not occur in refactoring. cc @YashKamboj

**Comment by leecalcote** [comment]
```
AssertionError: Timed out retrying after 4000ms: Expected to find element: `[data-cy="import-button"]`, but never found it.```

**Comment by Chadha93** [comment]
All the tests here are queued.... What could be the issue?? The runner is not picking up the jobs.

**Comment by leecalcote** [comment]
@Chadha93 looks like it finally ran:

```
AssertionError: Timed out retrying after 4000ms: Expected to find element: `[data-cy="import-button"]`, but never found it.
```

**Comment by Abhishek-kumar09** [comment]
SO filters test needs to be updated to pass the integration test. Seems like the api request is not sent on file upload. Any changes in that area by you? Previously, the import actions directly runs after selecting browse, probably you have progressed it make it better? that may be reason of fail.

**Comment by leecalcote** [comment]
Still failing - https://github.com/meshery/meshery/runs/7772269872?check_suite_focus=true

**Comment by MarioArriaga92** [comment]
@Chadha93 @leecalcote taking a look at failing test right now

**Comment by leecalcote** [comment]
Thank you. :)

**Comment by Chadha93** [comment]
Yay! All tests passed. Thank you @MarioArriaga92 @Abhishek-kumar09 
Merging this.

**Review by MarioArriaga92 — COMMENTED** [review]
LGTM

**Pull request merged**